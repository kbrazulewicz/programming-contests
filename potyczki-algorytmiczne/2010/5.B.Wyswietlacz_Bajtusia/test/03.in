14
4 2 81
3 2 42
3 2 62
2 2 54
3 2 87
2 2 84
2 3 841
2 3 864
2 3 804
3 3 662
4 3 662
3 3 462
4 6 866442
6 5 76221
