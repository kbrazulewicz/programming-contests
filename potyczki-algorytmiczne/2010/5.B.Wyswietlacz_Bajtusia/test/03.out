98
94
96
95
98
98
981
986
984
982
992
982
998662
99772
